services/cognito/app/resources/pool_resource.rb

```diff
@@ -13,4 +13,8 @@ class PoolResource < Cognito::ApplicationResource
     filter_fields = /\D/.match?(value[0]) ? query_by_non_id_attrs : query_by_id
     records.where(filter_fields)
   }
+
+  def users_count
```

Comment:
Also for a matter of consistency (since we return `record_count`) I think this should be `user_count`